README.md

```diff
@@ -2,6 +2,17 @@
 
 Below you can find and introduce yourself to the main aspects and technologies that you need to understand and cover if you are planning to start a career in QA Engineering.
 
+<details>
+<summary>Translations:</summary>
+
+:brazil: [Português Brasileiro](translations/README-ptbr.md)
+
+</details>
+
+<details>
+    <summary>Translations in progress:</summary>
+</details>
+
```

Comment:
Better to remove the `Translations` and `Translations in Progress` sections as the translations dir is clear enough.

Reply:
solved!